src/iuse.cpp

```diff
@@ -1073,7 +1104,10 @@ static int marloss_reject_mutagen( player *p, item *it )
         return 0;
     }
     if (p->has_trait("THRESH_MARLOSS")) {
-        p->add_msg_if_player(m_warning, _("The %s burns white-hot inside you, and you collapse to the ground!"), it->tname().c_str());
+        p->add_msg_if_player( m_warning,
```

Comment:
Currently yes.
Although I just noticed that this line (and few similar ones) is wrong anyway: that `add_msg_if_player` should be `add_msg_player_or_npc`.